applications/asset_tracker_v2/src/modules/Kconfig.app_module

```
@@ -8,7 +8,7 @@ menu "Application control module"
 
 config APP_REQUEST_GNSS_ON_INITIAL_SAMPLING
 	bool "Request GNSS on initial data sampling after connection"
-	default y if GPS_MODULE
+	default y if GNSS_MODULE
```

```suggestion
	default y if GPS_MODULE
```

Addressed in #6496, an-it's a non-related change.